secp256k1/src/main/java/org/hyperledger/besu/nativelib/secp256k1/LibSecp256k1.java

```diff
@@ -44,11 +44,14 @@ private static PointerByReference createContext() {
       Native.register(LibSecp256k1.class, "secp256k1");
       final PointerByReference context =
           secp256k1_context_create(SECP256K1_CONTEXT_VERIFY | SECP256K1_CONTEXT_SIGN);
-      if (secp256k1_context_randomize(context, new SecureRandom().generateSeed(32)) == 1) {
-        return context;
-      } else {
-        return null;
+      if (Boolean.parseBoolean(System.getProperty("secp256k1.randomize", "true"))) {
+        // randomization requested or not explicitly disabled
+        if (secp256k1_context_randomize(context, new SecureRandom().generateSeed(32)) != 1) {
+          // there was an error, don't preserve the context
+          return null;
```

Comment:
should this throw?

Reply:
No.  This is done during class initialization and either the `CONTEXT` has a value or is null, where null indicates native acceleration is not available.  If we throw an exception during class initialization then the whole startup of besu may abort.